docs/tutorial.rst

```diff
@@ -11,19 +11,19 @@ stochastic block model (SBM), degree-corrected SBM, and random dot product graph
 
 .. toctree::
    :maxdepth: 1
-      
+
```

Comment:
@dfrancisco1998  a lot of these changes also look unintentional?

Reply:
@bdpedigo yes they are... again i am sorry. I really do not know what is going on with all this. I will put it  back to the way it was (same method i used to get rid of all the weird changed that happening in plot.py).

Reply:
@bdpedigo I have gone through multiple times trying to fix these errors manually. But these spacing things will not save. And upon inspection (i redownloaded all the graspologic code) these spaces do not exist there either. So I am quite confused.

Reply:
can you expand on "will not save"

Reply:
@bdpedigo I made the edits for the spacing. Saved the .rst file. git add. git commit. git push. And it still says those spaces are missing.